libobs/util/platform-windows.c

```diff
@@ -1088,9 +1088,24 @@ bool is_64_bit_windows(void)
 #endif
 }
 
+bool is_arm64_windows(void)
+{
+#if defined(_M_ARM64) || defined(_M_ARM64EC)
+	return true;
+#else
+	USHORT _, nativeMachine;
```

Comment:
Please use a meaningful name. Please also do not start identifiers with an underscore, per the C standard section on reserved identifiers.